examples/simulation.yml

```diff
@@ -0,0 +1,89 @@
+# Whether to link or copy data
+copy_method: copy
+
+# Map each beamline to a session
+sessions:
+  bl: blc00001-1
+
+# Where to copy raw data from
+raw_data: /data/ispyb-test
+
+# Where to write simulated data to, can use {beamline} placeholder
+data_dir: /data/tests/{beamline}/simulation
+
+ispyb_url: https://ispyb.esrf.fr
+
+# Define Components (Proteins)
+components:
+  # an internal reference for the component
+  comp1:
+    # columns to populate for this component
+    acronym: Component1
+    sequence: SiSP
+    molecularMass: 12.5
+
+  comp2:
+    acronym: Component2
+
+# Define BLSamples
+samples:
+  # an internal reference for this sample
+  samp1:
+    # columns to populate for this sample
+    name: Sample1
+    # which component this sample is an instance of (one of the keys in components above)
+    component: comp1
+
+  samp2:
+    name: Sample2
+    component: comp2
+
+# Define Experiments (DataCollections)
+experiments:
+  # a shortname for this experiment (available via cli)
+  energy_scan1:
+    # the experimentType, must map to a valid type in DataCollectionGroup.experimentType
+    experimentType: OSC
+    # data will be split into its respective imageDirectory and fileTemplate columns
+    data: osc/oscillation.h5
+    # which sample to link this data collection to (one of the keys in samples above)
+    sample: samp1
+
+    # columns to populate
+    # thumbnails should have a trailing t, i.e. energy_scan/snapshott.png
```

Comment:
I guess here it would be better "# thumbnails should have a trailing t, i.e. osc/snapshot1t.png", since the line below has the fullsize image as "osc/snapshot1.png"

Fullsize image example: osc/snapshot1.png
Thumbnail example: osc/snapshot1t.png